pyrms/rms.py

```diff
@@ -28,3 +28,26 @@ def pygetfluxdiagram(bsol,t,centralspecieslist=[],superimpose=False,
         locals()[item] = getattr(ReactionMechanismSimulator,item)
     except AttributeError:
         pass
+
+# These functions force the returned object to be a jlwrap object, 
+# to avoid the solution object gets converted into list of lists
+_threadedsensitivities_inds = Main.pyfunctionret(ReactionMechanismSimulator.threadedsensitivities, Main.Any, Main.PyAny, Main.PyAny)
+_threadedsensitivities = Main.pyfunctionret(ReactionMechanismSimulator.threadedsensitivities, Main.Any, Main.PyAny)
+# Allow us the get the solution object in the julia wrapped object without
+# it being converted into list of lists
+get = Main.pyfunctionret(Main.get, Main.Any, Main.PyAny, Main.PyAny, Main.PyAny)
+def threadedsensitivities(react, inds=None):
+    if inds is not None:
+        # the sol_dict returned here is a jlwrap object and is not easy to access the contents
+        sol_dict = _threadedsensitivities_inds(react, inds)
+        # pysol_dict is the python dictionary, but the solution object got turned into list of lists
+        pysol_dict = Main.PyObject(sol_dict)
+        # this remakes the dictionary with the values as julia wrapped object ODEsolution, 
+        # but at the same time allows the user to access it normally like a dictionary
+        return {key: get(sol_dict, key, Main.Any) for key in pysol_dict.keys()}
+    else:
+        sol = _threadedsensitivities(react)
+        return sol
+
+ReactionMechanismSimulator.threadedsensitivities = pythreadedsensitivities
```

Comment:
Does pythreadedsensitivities get defined anywhere?  Also is it possible to move this up with pygetfluxdiagram?

Reply:
Thanks for catching this, the function was supposed to be `pythreadedsensitivities`. I also move this up with pygetfluxdiagram